packages/core-transaction-pool/lib/utils/dynamicfee-matcher.js

```diff
@@ -20,24 +20,25 @@ module.exports = (transaction) => {
   }
 
   if (feeConstants.dynamic) {
-    const calculatedFee = dynamicFeeManager.calculateFee(config.delegates.dynamicFees.feeMultiplier, transaction)
-
-    if (transactionFee < config.delegates.dynamicFees.minAcceptableFee) {
-      logger.debug(`Fee declined - Received transaction "${transaction.id}" with a fee of "${transactionFee}" which is below the minimum accepted fee of "${config.delegates.dynamicFees.minAcceptableFee}" by this delegate.`)
-      return false
-    }
-
-    if (calculatedFee > transactionFee) {
-      logger.debug(`Fee declined - Received transaction "${transaction.id}" with a fee of "${transactionFee}" which is below the calculated fee of "${calculatedFee}".`)
+    const minFeeFixed = config.delegates.dynamicFees.minAcceptableFee
+    if (transactionFee < minFeeFixed) {
+      logger.debug(
+        `Fee declined - Received transaction "${transaction.id}" with a fee of ` +
+        `"${transactionFee}" which is below the minimum accepted fixed fee of "${minFeeFixed}".`)
       return false
     }
 
-    if (transactionFee > staticFee) {
-      logger.debug(`Fee declined - Received transaction "${transaction.id}" with a fee of "${transactionFee}" which is higher than the static fee of "${feeManager.get(transaction.type)}".`)
+    const minFeeCalculated = dynamicFeeManager.calculateFee(config.delegates.dynamicFees.feeMultiplier, transaction)
```

Comment:
We dont have a check for user errors now?

Reply:
@kristjank https://github.com/ArkEcosystem/core/issues/1351#issuecomment-437453268